src/concuerror_callback.erl

```diff
@@ -93,6 +96,9 @@
           monitors                    :: monitors(),
           event = none                :: 'none' | event(),
           notify_when_ready           :: {pid(), boolean()},
+          number_of_processes         :: pos_integer(),
```

Comment:
This should be the responsibility of the generator and not part of the state of an instrumented process. When the limit is reached the answer can contain this number (I have left a comment at the other location).